l1-contracts/contracts/bridgehub/MessageRoot.sol

```diff
@@ -35,6 +35,8 @@ contract MessageRoot is IMessageRoot, Initializable {
 
     event Preimage(bytes32 one, bytes32 two);
 
+    event NewMessageRoot(uint256 indexed chainId, uint256 indexed blockNumber, uint256 indexed logId, bytes32[] sides);
```

Comment:
Why `sides` is `bytes32[]` and not `bytes32`?

Reply:
forward compatiblitilty

Reply:
There is a longer description in the MessageRoot struct